.github/ISSUE_TEMPLATE/feature_work_template.md

```diff
@@ -18,20 +18,12 @@ _As a [type of user], I want to [do something] [to produce this outcome]_
 -----------------------------------------
 
 ### Tasks
-- [ ] _Key pieces of work that will need to be done to meet the User Story and CoAs_
+#### UX
+
+#### Dev
 
 ### Definition of Done Checklist
-- [ ] Usability validated
-- [ ] UI works as designed
-- [ ] OESA: Code refactored for clarity
-- [ ] OESA: Dependency rules followed
-- [ ] Automated unit tests updated and passed
-- [ ] Automated integration tests updated and passed
-- [ ] Automated quality tests updated and passed
-- [ ] Automated load tests updated and passed
-- [ ] Automated a11y tests updated and passed
-- [ ] Automated security tests updated and passed
-- [ ] 90%+ Code coverage achieved
+- [ ] Usability validated (UX team)
 - [ ] PR(s) have been merged to main
```

Comment:
was hoping we could get some clarity on who checks these off per item. Maybe like Tech Lead, UX Team, IC

Reply:
@fpigeonjr yeah that's fair. I was of the belief that a developer would need to check almost all these expect the usability and UI ones, which is where I put the UX team in parenthetical.  So I guess inherently I was assuming to put in parentheticals any items were not a developer checking it off. But that may not be desirable. ??    As to whether the lead or a non-lead, I'm not sure.  That's something maybe that should be discussed in some venue.

Reply:
@tdonaworth @fpigeonjr @johndeange @stevtek @xlorepdarkhelm  What thoughts do you have on which, if any, items should be done by a lead vs. someone else ?

Reply:
I'm honestly not sure. There are checks (like the 90% code coverage reached, Automated <whatever> tests updated and passed, etc) which are largely done through the CICD system before allowing the code to be released. Things like OESA I can see the developer for whatever piece being responsible for, UI works as designed (and maybe usability validated?) I can see the UX team maybe needing to check that one. As far as a lead vs someone else.... not sure on that.

Reply:
If we move the UI review, to a PR basis, with our current deployment model, that might be problematic since we’d be expecting a UI review before it ever got (automatically) deployed to an environment. Hmmmm. Any thoughts ?

Reply:
> If we move the UI review, to a PR basis, with our current deployment model, that might be problematic since we’d be expecting a UI review before it ever got (automatically) deployed to an environment. Hmmmm. Any thoughts ?

@andressacurvelo @kimschulke Based on this note I'm quoting (also the one directly before this), I moved the UI review back to the story template instead of the PR, much to my chagrin.  Doing it on a per PR-basis would definitely simplify and make life easier but given our current branching and deployment model, I don't know that we can do it at this time unless there's a better option that presents itself. 

If that changes anything for anyone reading this who hasn't yet approved the PR such that you'd be inclined to approve or provide further commentary, please do so. Would love to discuss further

Reply:
I guess it depends if its a design/visual change or a content change. Content changes can easily be approved in a PR before we would actually see something in dev

Reply:
@kimschulke well, a user story is almost certainly going to have a UI/visual change but a given PR won't always necessarily.  Maybe that's the same thing as what you were saying, I'm not sure. :-)